README.md

```diff
@@ -6,37 +6,21 @@ An [Ice](https://github.com/zeroc-ice/ice) installation with `slice2java` versio
 
 The Ice Builder for Eclipse plug-in provides the following features:
 
-  - Handles all aspects of translating your Slice files
+  - Automatically compiles all the Slice files in your project
   - Incrementally recompiles Slice files after modifications
   - Maintains dependencies between Slice files
   - Highlights compilation errors in your source code
   - Manages the generated code to remove obsolete files automatically
 
 ## Contents
 
-- [Build Instructions](#build-instructions)
-- [Usage](#usage)
+- [Usage](#Usage)
   - [Installing the Plug-in](#installing-the-plug-in)
   - [Configuring the Plug-in](#configuring-the-plug-in)
   - [Activating the Plug-in for a Project](#activating-the-plug-in-for-a-project)
   - [Configuring Project Settings](#configuring-project-settings)
-    - [Settings in the Source Tab](#settings-in-the-source-tab)
-    - [Settings in the Options Tab](#settings-in-the-options-tab)
-  - [Configuring File Settings](#configuring-file-settings)
-- [When does the Plug-In Recompile Slice Files?](#when-does-the-plug-in-recompile-slice-files)
-
-## Build Instructions
-
-In order to build the plug-in, you will need an installation of Eclipse suitable for plug-in development.
-
-It will be necessary to install the Eclipse Plug-in Development Environment
-if it is not already installed. Go to `Help > Eclipse Marketplace` and search
-for PDE, then click install to install the Eclipse PDE.
-
-To import the project choose `File > Import > General > Existing Projects into Workspace`
-and select the java directory in this repository as the root directory.
-
-To create the plug-in, use `File > Export > Plug-in Development > Deployable plug-ins and fragments`.
+  - [Using the Plug-in](#using-the-plugin)
+- [Building the Plug-in from Source](#building-the-plugin-from-source)
 
 ## Usage
 
```

Comment:
Leave it as an exercise I'd say. Seems outside of the scope of the Builder to me.